.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,30 +1,25 @@
-name: "⭐ Feature / enhancement request"
+name: "⭐ New feature request"
```

Comment:
Revert the changes in this file.  I don't want feature/enhancement seperated

Reply:
agree